private_set_intersection/javascript/src/implementation/client.ts

```diff
@@ -104,10 +132,11 @@ export const ClientWrapperConstructor = ({
      *
      * @function
      * @name Client.create
+     * @param {boolean} [revealIntersection=false] - If true, reveals the actual intersection instead of the cardinality.
      * @returns {Client} A Client instance
      */
-    create(): Client {
-      const { Value, Status } = library.PsiClient.Create()
+    create(revealIntersection = false): Client {
```

Comment:
We should be consistent across language bindings, either provide default values for all of them or for none of them.
For the C++ version I chose not to have a default value for reveal_intersection so users have to think about what variant they want. But if you feel that the cardinality version should be the default, then we can add the default value everywhere.

Reply:
Most JS libraries have defaults to minimize runtime errors for initialization and adhere as best as possible to zero-config. 

In this case, I was trying to output the cardinality and not the intersection a "better" default configuration.

If this is not typical in other interpreted languages, I'll take it out.